std/experimental/allocator/building_blocks/region.d

```diff
@@ -81,7 +79,7 @@ struct Region(ParentAllocator = NullAllocator,
         $(D parent.allocate(n)) returns $(D null), the region will be initialized
         as empty (correctly initialized but unable to allocate).
         */
-    this(ubyte[] store)
+    this(ubyte[] store) pure nothrow @nogc
```

Comment:
You are right that `this(ubyte[] store)` should not be trusted.

The problem with it is that if you use the GC to provide the store,
you can not allocate objects that store pointers to other GC allocated objects
because the GC will not perform a scan on it. Please see #5322 for more details.

I've read your [forum post](https://forum.dlang.org/thread/jfgpngdudtprzznrckwf@forum.dlang.org?page=1)
and I am afraid that you might get bit by this issue, so please use with care until we get the fix in.
I've made a few attempts to fix this, see #5403 and #5882, but those have hit a stall.

@andralex and I have talked about this issue on a regular basis and we are thinking of a solution.

Reply:
So which of the attributes `pure nothrow @nogc` should I remove then?